pipeline.go

```diff
@@ -95,7 +96,15 @@ func clearSetup() error {
 	return nil
 }
 
-// Run executes the pipeline.
+// Run executes the pipeline. For each stage defined in the pipeline we execute
+// the `docker build` and `docker run`. If any of these two processes fail,
+// we interrupt the flow and the error handler is called. The error handler is a
+// stage like any other, but it is only built and executed in case of an error in
+// the standard flow. If there are any errors in the `docker build` and `docker run`
+// of the error handler, the process is stopped and the error is returned.
+// If a specific error handler has not been defined, the default behavior is to
```

Comment:
Por retorno você quis dizer imprimir na saída padrão? Imprimir na saída de erro?

Reply:
Imprimir na saída de erro:
`return *result, fmt.Errorf("error when building image for error handler: %s", err)`
(E suas variantes no caso dos demais erros possíveis dentro da execução de um estágio)

ou

`return *result, fmt.Errorf(msg)`
(Em caso de um error handler não ter sido definido.)